modules/increasing-aws-flavor-size.adoc

```diff
@@ -16,7 +17,7 @@ It is recommended to backup etcd before increasing the flavor size of the AWS ma
 .Prerequisites
 
 * You have an IPI (installer-provisioned infrastructure) or UPI (user-provisioned infrastructure) cluster on AWS.
-
+//todo: assess for CPMS impact (see: https://issues.redhat.com/browse/OSDOCS-4622)
```

Comment:
Scalability and performance > Recommended host practices > [Increasing the flavor size of the Amazon Web Services (AWS) master instances](https://53397--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/recommended-host-practices.html#increasing-aws-flavor-size_recommended-host-practices)

Reply:
The IMPORTANT admonition in the section prior to this says IPI control planes can't be edited. This is not true for clusters with CMPS. For this prerequisite, I think: 
- change it to say UPI only, since this procedure is not using the CMPS
- add an "Additional Resources" link to CMPS docs for IPI clusters that support it
- what do we say about IPI clusters that don't have the CPMS yet?

Reply:
IPI clusters that don't have CPMS should remain with the existing note that the control plane should not be edited right? We should prompt users maybe to configure CPMS on supported platforms?

Reply:
Ok I _think_ I have untangled all of this in the most recent push